README.md

```diff
@@ -68,9 +73,6 @@ This project can be utilized one of two ways, via docker or via a Makefile inclu
   # tardigrade-ci
   .tardigrade-ci
   tardigrade-ci/
-
-  # eclint
-  .git/
   ```
 
 4. Run `make` once to initialize the project and then `make docker/run target=<TARGET>`
```

Comment:
Instead of `make`, should this be `make init`?

Reply:
Two different things you're seeing, I think... If the project has not been init'd, then `make` alone will run the `init` target. After that, the tardigrade-ci/Makefile will be available and `make` will run the `help` target.

Reply:
Updated to `make init`